content/healthchecks/deploy.files/readiness-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
```

Comment:
Based on other suggestion in readiness.md, move this file to `content/healthchecks/readiness.files/readiness-deployment.yaml`